frontend/app/src/store/session/index.ts

```diff
@@ -236,7 +226,6 @@ export const useSessionStore = defineStore('session', () => {
 
   async function cleanup() {
     stop();
-    reset();
```

Comment:
Since cleanup has only one usage we can inline it in the place where it is used.

Also, we need to add this just after `stop`, otherwise, the logout cleanup won't trigger.
```typescript
      set(logged, false);
```

Reply:
https://github.com/rotki/rotki/blob/eeae5057744f1a54cf388fc1925a77b738002e80/frontend/app/src/composables/logout.ts#L7